### ISSUE TITLE
Smart sorting the results

### ISSUE THREAD

**Issue by gyorb** [Ericsson/codechecker#296]
Smart sorting could sort the results based on:
* severity
* bugpath length
* alpha or not alpha checker
* checker category
* ...

This could help to move the most important results to the top of the list.

**Comment by bruntib** [comment]
Is this ticket about sorting automatically? If not, then it could be closed I think.

**Comment by gyorb** [comment]
Yes the idea here was at that time that if the user wants to start to cleanup the results with this smart sorting it would be easier to get a list of reports which should be fixed first.
Like a high severity report where the bug path is short which is probably easier to fix.

With the improved filtering most of the things can be done manually now.

**Comment by bruntib** [comment]
The current way of working is that users are selecting the reports in question, either through a statistics page (based on severities, guidelines, etc.) because it provides the set of reports they need to fix. Sorting them after the selection is working, so the required information (i.e. low-hanging fruits) can be selected easily. For this reason I would close this ticket, because providing a more improved sorting is not in our scope. However, if there is an idea on a different selection method, then feel free to comment. Thank you!
